packages/story-editor/src/components/header/buttons/test/update.js

```diff
@@ -84,7 +84,7 @@ function arrange({
 }
 
 describe('UpdateButton', () => {
-  const FUTURE_DATE = '2022-01-01T20:20:20Z';
+  const FUTURE_DATE = '2022-01-10T20:20:20Z';
```

Comment:
Maybe change the year here instead? Seem like this might not be accurate quite soon.

Reply:
I dont think it would matter because `Mockdate` actually manipulates the `now` date so anytime test is run the system is at `2020-07-15T12:00:00+00:00`

Reply:
Ah, okay, why the change at all then?

Reply:
sorry forgot to remove thought it was the reason the test failed will revert the change